db/buckets_test.go

```diff
@@ -1,17 +1,31 @@
 package db_test
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/NethermindEth/juno/db"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestKey(t *testing.T) {
-	key := db.StateRootKey.Key([]byte{1})
-	assert.Equal(t, []byte{byte(db.StateRootKey), 1}, key)
-	key = db.StateRootKey.Key([]byte{1}, []byte{2})
-	assert.Equal(t, []byte{byte(db.StateRootKey), 1, 2}, key)
-	key = db.StateTrie.Key([]byte{1}, []byte{2})
-	assert.Equal(t, []byte{byte(db.StateTrie), 1, 2}, key)
+	t.Run("bucket with no key", func(t *testing.T) {
+		key := db.StateRootKey.Key()
+		assert.Equal(t, []byte{byte(db.StateRootKey)}, key)
+	})
+	t.Run("bucket with nil key", func(t *testing.T) {
+		key := db.StateRootKey.Key(nil)
+		assert.Equal(t, []byte{byte(db.StateRootKey)}, key)
+	})
+	t.Run("bucket with multiple keys", func(t *testing.T) {
+		keys := [][]byte{{}, {0}, {0, 1, 2, 3, 4}, {0, 1, 2, 3, 4, 5, 6, 7, 8, 9}}
+
+		for _, k := range keys {
+			t.Run(fmt.Sprint(len(k)), func(t *testing.T) {
```

Comment:
Minor nitpick. (Uber-go [recommendation](https://github.com/uber-go/guide/blob/master/style.md#prefer-strconv-over-fmt))
```suggestion
			t.Run(string(rune(len(k))), func(t *testing.T) {
```